Client/Frontend/Home/ActivityStreamTopSitesCell.swift

```diff
@@ -61,7 +61,8 @@ class TopSiteItemCell: UICollectionViewCell {
 
     override init(frame: CGRect) {
         super.init(frame: frame)
-
+        isAccessibilityElement = true
+        accessibilityIdentifier = "TopSite"
```

Comment:
I need a way of getting all of the TopSiteCells. Because the labels are based on the url its harder to query them. By setting the identifier to be Topsite I can easily do "cellsWithID(TopSite)" and get all the cells.